emr52/build.xml

```diff
@@ -0,0 +1,15 @@
+<!--===========================================================================
+  This is the build file for the Hadoop EMR 4.$ Shim modules.
+
+  See ../build-res/subfloor.xml for more details
+============================================================================-->
+<project name="pentaho-hadoop-shims-emr52" basedir="." default="dist"
```

Comment:
If this was <project name'${ivy.artifact.id}" ... You wouldn't have to change it every time.